### PR TITLE
3.0.0: Add changelog

### PR DESCRIPTION
**Pull request by GaryJones**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- [#786](https://github.com/Automattic/VIP-Coding-Standards/pull/786): Add PHPStan to QA checks.
+- [#787](https://github.com/Automattic/VIP-Coding-Standards/pull/787): GH Actions: tweak the way the PHPCS/WPCS versions are set.
+- [#789](https://github.com/Automattic/VIP-Coding-Standards/pull/789): Updates related to branch rename from `master` to `main`.
+- [#790](https://github.com/Automattic/VIP-Coding-Standards/pull/790): PHPUnit: Use 7.5 schema.
+- [#791](https://github.com/Automattic/VIP-Coding-Standards/pull/791): Docs: Update `CONTRIBUTING.md`.
+
+### Removed
+- [#774](https://github.com/Automattic/VIP-Coding-Standards/issues/774): Performance/BatcacheWhitelistedParams: remove the sniff.
+- [#775](https://github.com/Automattic/VIP-Coding-Standards/issues/775): Compatibility/Zoninator: remove the sniff.
+- [#776](https://github.com/Automattic/VIP-Coding-Standards/issues/776): Variables/VariableAnalysis: remove the sniff.
```
Comment (jrfnl) [inline CHANGELOG.md:41]:
```suggestion
- [#776](https://github.com/Automattic/VIP-Coding-Standards/pull/776): Variables/VariableAnalysis: remove the sniff.
    This VIPCS native version of the sniff was previously already replaced by the more comprehensive external `VariableAnalysis` standard.
```

Re: the link - underlying issue would be #496 if you prefer to link to the issue.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- [#785](https://github.com/Automattic/VIP-Coding-Standards/pull/785): Docs: remove redundant `@package` tags.
+- [#786](https://github.com/Automattic/VIP-Coding-Standards/pull/786): Add PHPStan to QA checks.
+- [#787](https://github.com/Automattic/VIP-Coding-Standards/pull/787): GH Actions: tweak the way the PHPCS/WPCS versions are set.
+- [#789](https://github.com/Automattic/VIP-Coding-Standards/pull/789): Updates related to branch rename from `master` to `main`.
+- [#790](https://github.com/Automattic/VIP-Coding-Standards/pull/790): PHPUnit: Use 7.5 schema.
+- [#791](https://github.com/Automattic/VIP-Coding-Standards/pull/791): Docs: Update `CONTRIBUTING.md`.
+
+### Removed
+- [#774](https://github.com/Automattic/VIP-Coding-Standards/issues/774): Performance/BatcacheWhitelistedParams: remove the sniff.
+- [#775](https://github.com/Automattic/VIP-Coding-Standards/issues/775): Compatibility/Zoninator: remove the sniff.
```
Comment (jrfnl) [inline CHANGELOG.md:40]:
```suggestion
- [#774](https://github.com/Automattic/VIP-Coding-Standards/pull/774): Performance/BatcacheWhitelistedParams: remove the sniff.
- [#775](https://github.com/Automattic/VIP-Coding-Standards/pull/775): Compatibility/Zoninator: remove the sniff.
```

Or link to the underlying issues - #612 for Zoninator, #613 for Batcache.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- [#744](https://github.com/Automattic/VIP-Coding-Standards/issues/744): WPCS: 3.0 `merge_custom_array()`.
+- [#777](https://github.com/Automattic/VIP-Coding-Standards/pull/777): 3.0: start using PHPCSUtils.
+- [#779](https://github.com/Automattic/VIP-Coding-Standards/pull/779): 3.0: support WordPressCS 3.0.
+
+### Maintenance
+- [#778](https://github.com/Automattic/VIP-Coding-Standards/pull/778): CS: improve use statements.
+- [#780](https://github.com/Automattic/VIP-Coding-Standards/pull/780): Performance/WPQueryParams: defer to the parent sniff.
+  - Two error codes changed:
+    - `WordPressVIPMinimum.Performance.WPQueryParams.PostNotIn` is now `WordPressVIPMinimum.Performance.WPQueryParams.PostNotIn_post__not_in`.
+    - `WordPressVIPMinimum.Performance.WPQueryParams.SuppressFiltersTrue` is now `WordPressVIPMinimum.Performance.WPQueryParams.SuppressFilters_suppress_filters`.
```
Comment (jrfnl) [inline CHANGELOG.md:27]: Should this be in a "Changed" section ? (as it is a functional change with a BC break)

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+This release requires [WordPressCS 3.0.0](https://github.com/WordPress/WordPress-Coding-Standards/releases/tag/3.0.0). It is not compatible with WordPressCS 2.x. Users should read the [WordPressCS 3.0 upgrade guide for end-users](https://github.com/WordPress/WordPress-Coding-Standards/wiki/Upgrade-Guide-to-WordPressCS-3.0.0-for-ruleset-maintainers).
+
+Increases requirements for PHPCS from 3.7.1 to 3.7.2.
+
+### Fixed
+- [#784](https://github.com/Automattic/VIP-Coding-Standards/pull/784): Performance/WPQueryParams: prevent false positives for `'exclude'` with `get_users()`.
+- [#788](https://github.com/Automattic/VIP-Coding-Standards/pull/788): Security/Mustache: prevent false positives on block editor templates.
+
+### WordPressCS 3.0 Compatibility
+- [#744](https://github.com/Automattic/VIP-Coding-Standards/issues/744): WPCS: 3.0 `merge_custom_array()`.
```
Comment (jrfnl) [inline CHANGELOG.md:18]: This was fixed as part of #779. Does this warrant an explicit separate entry ?

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+
+This release requires [WordPressCS 3.0.0](https://github.com/WordPress/WordPress-Coding-Standards/releases/tag/3.0.0). It is not compatible with WordPressCS 2.x. Users should read the [WordPressCS 3.0 upgrade guide for end-users](https://github.com/WordPress/WordPress-Coding-Standards/wiki/Upgrade-Guide-to-WordPressCS-3.0.0-for-ruleset-maintainers).
+
+Increases requirements for PHPCS from 3.7.1 to 3.7.2.
+
+### Fixed
+- [#784](https://github.com/Automattic/VIP-Coding-Standards/pull/784): Performance/WPQueryParams: prevent false positives for `'exclude'` with `get_users()`.
+- [#788](https://github.com/Automattic/VIP-Coding-Standards/pull/788): Security/Mustache: prevent false positives on block editor templates.
+
+### WordPressCS 3.0 Compatibility
```
Comment (jrfnl) [inline CHANGELOG.md:17]: Should this be "Added" (and at the top as the most important change) ?

**Review by jrfnl — COMMENTED** [review]
@GaryJones Thanks for setting this up.

Verified that all merged PRs are mentioned ✔️ 

Additional notes:
* "Props" are missing. Not something I'm really concerned about as it's only the two of us, but all previous changelogs have them, so for consistency, it should be there.
* I would mention the branch "rename" in the intro at the top as well. 
    Also: the branch rename has not been done in a typical _rename_ manner, but has _added_ a `main` branch instead. This is good in that it won't accidentally break CI for any repo now using `master`, but that makes it all the more important to signal that `main` should be used from now on.
* In most cases, the issue number/link goes to the PR, though sometimes to an underlying issue.
    Would it be an idea to use a consistent pattern ?
    Maybe something like `[PR link]: description. Fixes [issue link], [issue link]` ?

**Comment by GaryJones** [comment]
@jrfnl PR updated with your suggestions :-)

**Review by jrfnl — APPROVED** [review]

**Pull request merged**